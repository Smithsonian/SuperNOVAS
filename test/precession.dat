
2450545.000000 TEST       S2 O0 A1: 33579669.044179 -19361996.966670 14118275.822216 

2450545.000000 TEST       S2 O1 A1: 33579669.044179 -19361996.966670 14118275.822216 

2450545.000000 TEST       S2 O2 A1: 33579669.044179 -19361996.966670 14118275.822216 

2450545.000000 SUN        S0 O0 A1:    -0.006754     0.004991     0.002337 

2450545.000000 SUN        S0 O1 A1:    -0.006754     0.004991     0.002337 

2450545.000000 SUN        S0 O2 A1:    -0.006754     0.004991     0.002337 

2451545.000000 TEST       S2 O0 A1: 33571564.213573 -19382551.635823 14109343.719135 

2451545.000000 TEST       S2 O1 A1: 33571564.213573 -19382551.635823 14109343.719135 

2451545.000000 TEST       S2 O2 A1: 33571564.213573 -19382551.635823 14109343.719135 

2451545.000000 SUN        S0 O0 A1:    -0.007138    -0.002644    -0.000921 

2451545.000000 SUN        S0 O1 A1:    -0.007138    -0.002644    -0.000921 

2451545.000000 SUN        S0 O2 A1:    -0.007138    -0.002644    -0.000921 

2452545.000000 TEST       S2 O0 A1: 33563444.184984 -19403101.678119 14100413.878894 

2452545.000000 TEST       S2 O1 A1: 33563444.184984 -19403101.678119 14100413.878894 

2452545.000000 TEST       S2 O2 A1: 33563444.184984 -19403101.678119 14100413.878894 

2452545.000000 SUN        S0 O0 A1:     0.000653    -0.004705    -0.002013 

2452545.000000 SUN        S0 O1 A1:     0.000653    -0.004705    -0.002013 

2452545.000000 SUN        S0 O2 A1:     0.000653    -0.004705    -0.002013 
