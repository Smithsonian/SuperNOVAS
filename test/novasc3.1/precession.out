
-10000.0 22+20      S2 O0 A0:  0.815745757 -0.464858709  0.344187798 

-10000.0 22+20      S2 O1 A0:  0.815745757 -0.464858709  0.344187798 

-10000.0 22+20      S2 O2 A0:  0.815745757 -0.464858709  0.344187798 

-10000.0 16-20      S2 O0 A0: -0.463944364 -0.816655773 -0.343262256 

-10000.0 16-20      S2 O1 A0: -0.463944364 -0.816655773 -0.343262256 

-10000.0 16-20      S2 O2 A0: -0.463944364 -0.816655773 -0.343262256 

-10000.0 SUN        S0 O0 A0: -0.008899235  0.916055482  0.400952811 

-10000.0 SUN        S0 O1 A0: -0.008899235  0.916055482  0.400952811 

-10000.0 SUN        S0 O2 A0: -0.008899235  0.916055482  0.400952811 

     0.0 22+20      S2 O0 A0:  0.813797681 -0.469846310  0.342020143 

     0.0 22+20      S2 O1 A0:  0.813797681 -0.469846310  0.342020143 

     0.0 22+20      S2 O2 A0:  0.813797681 -0.469846310  0.342020143 

     0.0 16-20      S2 O0 A0: -0.469846310 -0.813797681 -0.342020143 

     0.0 16-20      S2 O1 A0: -0.469846310 -0.813797681 -0.342020143 

     0.0 16-20      S2 O2 A0: -0.469846310 -0.813797681 -0.342020143 

     0.0 SUN        S0 O0 A0: -0.930944070 -0.344822002 -0.120170400 

     0.0 SUN        S0 O1 A0: -0.930944070 -0.344822002 -0.120170400 

     0.0 SUN        S0 O2 A0: -0.930944070 -0.344822002 -0.120170400 

 10000.0 22+20      S2 O0 A0:  0.811812765 -0.474822696  0.339857974 

 10000.0 22+20      S2 O1 A0:  0.811812765 -0.474822696  0.339857974 

 10000.0 22+20      S2 O2 A0:  0.811812765 -0.474822696  0.339857974 

 10000.0 16-20      S2 O0 A0: -0.475728039 -0.810903045 -0.340762504 

 10000.0 16-20      S2 O1 A0: -0.475728039 -0.810903045 -0.340762504 

 10000.0 16-20      S2 O2 A0: -0.475728039 -0.810903045 -0.340762504 

 10000.0 SUN        S0 O0 A0: -0.008957310 -0.923465112 -0.383577832 

 10000.0 SUN        S0 O1 A0: -0.008957310 -0.923465112 -0.383577832 

 10000.0 SUN        S0 O2 A0: -0.008957310 -0.923465112 -0.383577832 

-10000.0 22+20      S2 O0 A1:  0.815746 -0.464859  0.344188 

-10000.0 22+20      S2 O1 A1:  0.815746 -0.464859  0.344188 

-10000.0 22+20      S2 O2 A1:  0.815746 -0.464859  0.344188 

-10000.0 16-20      S2 O0 A1: -0.463944 -0.816656 -0.343262 

-10000.0 16-20      S2 O1 A1: -0.463944 -0.816656 -0.343262 

-10000.0 16-20      S2 O2 A1: -0.463944 -0.816656 -0.343262 

-10000.0 SUN        S0 O0 A1: -0.008899  0.916055  0.400953 

-10000.0 SUN        S0 O1 A1: -0.008899  0.916055  0.400953 

-10000.0 SUN        S0 O2 A1: -0.008899  0.916055  0.400953 

     0.0 22+20      S2 O0 A1:  0.813798 -0.469846  0.342020 

     0.0 22+20      S2 O1 A1:  0.813798 -0.469846  0.342020 

     0.0 22+20      S2 O2 A1:  0.813798 -0.469846  0.342020 

     0.0 16-20      S2 O0 A1: -0.469846 -0.813798 -0.342020 

     0.0 16-20      S2 O1 A1: -0.469846 -0.813798 -0.342020 

     0.0 16-20      S2 O2 A1: -0.469846 -0.813798 -0.342020 

     0.0 SUN        S0 O0 A1: -0.930944 -0.344822 -0.120170 

     0.0 SUN        S0 O1 A1: -0.930944 -0.344822 -0.120170 

     0.0 SUN        S0 O2 A1: -0.930944 -0.344822 -0.120170 

 10000.0 22+20      S2 O0 A1:  0.811813 -0.474823  0.339858 

 10000.0 22+20      S2 O1 A1:  0.811813 -0.474823  0.339858 

 10000.0 22+20      S2 O2 A1:  0.811813 -0.474823  0.339858 

 10000.0 16-20      S2 O0 A1: -0.475728 -0.810903 -0.340763 

 10000.0 16-20      S2 O1 A1: -0.475728 -0.810903 -0.340763 

 10000.0 16-20      S2 O2 A1: -0.475728 -0.810903 -0.340763 

 10000.0 SUN        S0 O0 A1: -0.008957 -0.923465 -0.383578 

 10000.0 SUN        S0 O1 A1: -0.008957 -0.923465 -0.383578 

 10000.0 SUN        S0 O2 A1: -0.008957 -0.923465 -0.383578 
